modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SelectTypeAndSubtype.js

```diff
@@ -15,8 +15,8 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 
-import TreeFilter from './components/TreeFilter';
-import {nodeTreeArrayMapper} from './utils/tree-utils';
+import {nodeTreeArrayMapper} from '../utils/tree-utils';
```

Comment:
@claraizquierdo can you please:
- move the tree utils to `/components/TreeFilter/utils`?
- rename the file to camelcase `treeUtils.js`, as it should be when I created it?
I wanted to do this for a while and I always forgot :(
Txs!

Reply:
Ok Beltrán, I agree with you :)